### PR TITLE
Build changed directories only

### PR DESCRIPTION
**Pull request by lypnol**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# set -ev
+declare a CHANGED
+
+for f in `git --no-pager diff --raw master..${TRAVIS_COMMIT} | awk 'NF>1{print $NF}'`
```
Comment (david-ds) [inline build.sh]: it won't work for master no ?

Comment (david-ds) [reply build.sh]: and you should use `git diff branch1...branch2` with 3 dots

**Review by david-ds — CHANGES_REQUESTED** [review]

File: build.sh (hunk 1 above)
Comment (david-ds) [reply build.sh]: and use the `--name-only` flag of `git diff` instead of `awk`

Comment (lypnol) [reply build.sh]: what if we push to master directly ? branch1 == branch2

**Pull request merged**